specification/columns/skupricedetails.md

```diff
@@ -0,0 +1,54 @@
+# SKU Price Details
+
+The **SKU Price Details** column represents a list of relevant properties shared by all resources having the same [**SKU Price ID**](#skupriceid). These properties enable a practitioner to more easily calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics. They also help a practitioner understand the specifics of a **SKU Price ID**.
+
+
+The _SkuPriceDetails_ column adheres to the following requirements:
+
+* The _SkuPriceDetails_ column MUST be in [_KeyValueFormat_](#key-valueformat).
+* The key for a property SHOULD be formatted in [PascalCase](#glossary:pascalcase).
+* The properties (both keys and values) contained in the _SkuPriceDetails_ column MUST be shared across all charges having the same _SkuPriceId_.
+* The key for a property SHOULD remain consistent across comparable SKUs having that property and the values for this key SHOULD remain in a consistent format.
+* The _SkuPriceDetails_ column SHOULD NOT contain properties which are not applicable to the corresponding _SkuPriceId_.
+* The _SkuPriceDetails_ column MAY contain properties which are already captured in other dedicated columns.
+* Properties in _SkuPriceDetails_ which have a numerical value MUST represent the value for a the individual [_ConsumedUnit_](#consumedunit) i.e. a [_ConsumedQuantity_](#consumedquantity) of 1.
```

Comment:
There's a typo with "a the" in here. I still feel like it's a bit confusing so not sure how to change that. I feel like this is more straightforward. I don't have a blocking issue with this text. I just feel it could be a little clearer. Feel free to resolve if it's just me.
```suggestion
* Numeric properties in _SkuDetails_ MUST represent the value for a single [_ConsumedUnit_](#consumedunit).
```

Reply:
A property is the key and value, which is why I was specifying the property's value as numeric. I like the way you've phrased it, so here's my attempt.
"If a property has a numeric value, it must represent the value for a single [_ConsumedUnit_](#consumedunit)."